docs/ops/movement/StridedSlice_1.md

```diff
@@ -51,21 +50,83 @@
-           Negative values mean indexing starts from the end. For example, if `foo=[1,2,3]`, `begin[0]=-1` means `begin[0]=3`.
+*   **2**: `begin` - 1D tensor of type `T_IND` with begin indexes for input tensor slicing. **Required.**  
+    Out-of-bounds values are silently clamped. If `begin_mask[i]` is `1`, the value of `begin[i]` is ignored and the range of the appropriate dimension starts from `0`. Negative values mean indexing starts from the end. For example, if `data=[1,2,3]`, `begin[0]=-1` means `begin[0]=3`.
 
-*   **3**: `end` input - 1D input tensor with end indexes for input tensor slicing. Required.
-           Out-of-bounds values will be silently clamped. If `end_mask[i]` is 1, the value of `end[i]` is ignored
-           and the full range of the appropriate dimension is used instead.
-           Negative values mean indexing starts from the end. For example, if `foo=[1,2,3]`, `end[0]=-1` means `end[0]=3`.
+*   **3**: `end` - 1D tensor of type `T_IND` with end indexes for input tensor slicing. **Required.**  
+    Out-of-bounds values will be silently clamped. If `end_mask[i]` is `1`, the value of `end[i]` is ignored and the full range of the appropriate dimension is used instead. Negative values mean indexing starts from the end. For example, if `data=[1,2,3]`, `end[0]=-1` means `end[0]=3`.
 
-*   **4**: `stride` input - 1D input tensor with strides. Optional.
+*   **4**: `stride` - 1D tensor of type `T_IND` with strides. **Optional.**
 
-**Example**
+**Types**
+* *T*: any supported type.
+* *T_IND*: any supported integer type.
+
+**Example**  
+Example of `begin_mask` & `end_mask` usage.  
+```xml
+<layer ... type="StridedSlice" ...>
+    <data begin_mask="0,1,1" ellipsis_mask="0,0,0" end_mask="1,1,0" new_axis_mask="0,0,0" shrink_axis_mask="0,0,0"/>
+    <input>
+        <port id="0">
+            <dim>2</dim>
+            <dim>3</dim>
+            <dim>4</dim>
+        </port>
+        <port id="1">
+            <dim>2</dim> <!-- begin: [1, 0, 0] -->
+        </port>
+        <port id="2">
+            <dim>2</dim> <!-- end: [0, 0, 2] -->
+        </port>
+        <port id="3">
+            <dim>2</dim> <!-- stride: [1, 1, 1] -->
+        </port>
+    </input>
+    <output>
+        <port id="4">            
+            <dim>1</dim>
+            <dim>3</dim>
+            <dim>2</dim>
+        </port>
+    </output>
+</layer>
+```
+
+Example of `new_axis_mask` usage.
+```xml
+<layer ... type="StridedSlice" ...>
+    <data begin_mask="0,1,1" ellipsis_mask="0,0,0" end_mask="0,1,1" new_axis_mask="1,0,0" shrink_axis_mask="0,0,0"/>
+    <input>
+        <port id="0">
+            <dim>2</dim>
+            <dim>3</dim>
+            <dim>4</dim>
+        </port>
+        <port id="1">
+            <dim>2</dim>
+        </port>
+        <port id="2">
+            <dim>2</dim>
+        </port>
+        <port id="3">
+            <dim>2</dim>
+        </port>
+    </input>
```

Comment:
Add a comment what values are in these tensors. Otherwise it is not clear how the operation works.

Reply:
It is not important what values are there because of `begin_mask` & `end_mask` values they are ignored.